common/common-db/src/main/java/org/ow2/proactive/db/TransactionHelper.java

```diff
@@ -0,0 +1,123 @@
+package org.ow2.proactive.db;
+
+import org.objectweb.proactive.utils.Sleeper;
+import org.ow2.proactive.scheduler.core.properties.PASchedulerProperties;
+import org.apache.log4j.Logger;
+import org.hibernate.HibernateException;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.exception.LockAcquisitionException;
+
+
+public class TransactionHelper {
+
+    private static final Logger logger = Logger.getLogger(TransactionHelper.class);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename the "logger" logger to comply with the format "LOG(?:GER)?". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1312)